packages/build/test/integration/scripts.integration.js

```diff
@@ -266,6 +266,18 @@ describe('mocha', function() {
     );
   });
 
+  it('honors --lang option', () => {
+    const LANG = process.env.LANG;
+    const run = require('../../bin/run-mocha');
+    const command = run(
+      ['node', 'bin/run-mocha', '--lang', 'fr', '"dist/__tests__"'],
+      true,
+    );
+    assert.equal(process.env.LANG, 'fr');
+    assert(command.indexOf('--lang fr') === -1, '--lang fr should be removed');
+    process.env.LANG = LANG;
```

Comment:
When one of the assertions above fail (throw an error), this line is not executed and subsequent tests may start failing because of a wrong locale, which is difficult to debug.

A more robust solution is to move `process.env.LANG` manipulation to `beforeEach` and `afterEach` hooks.

Reply:
I was aware of the possibility but didn't bother to go far enough.